pkg/prometheus/operator.go

```diff
@@ -1834,6 +1834,10 @@ func (c *Operator) selectProbes(p *monitoringv1.Prometheus) (map[string]*monitor
 			)
 			continue
 		}
+		pnKey := fmt.Sprintf("probe/%s/%s/%d", probe.GetNamespace(), probe.GetName())
+		if err = store.AddBearerToken(ctx, probe.GetNamespace(), probe.Spec.BearerTokenSecret, pnKey); err != nil {
```

Comment:
I think this needs something similar to what's in the selectservicemonitor and podmonitor functions for TLS assets to also be selected